### PR TITLE
[DOC] Update of grafana sections

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

**Documentation**

Updates to the Grafana documentation in the _Deploying_ guide

- Removed 1 file: include::modules/con_metrics-grafana-options.adoc[leveloffset=+1] to consolidate content
- Removed 4 screen captures that provided little benefit
- Made Grafana and Exporter sections consistent
- Listed dashboard artifacts -- this could be used instead of screen captures
- Note on Route option for Openshift

There were two dashboard screen captures in the guide. Because of their size, it was hard to see what was represented. I've kept the Kafka dashboard to show what they look like, but listed what is shown in the other dashboards. If the lists are too much, we could reduce them. 

Maybe we want more info on how to interpret the charts

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc [hunk 1]
```diff
@@ -23,20 +24,29 @@ If you are accessing the user interface for the first time, see xref:assembly-me
 
 .Procedure
 
-. Access the Grafana user interface.
+. xref:proc-metrics-grafana-dashboard-{context}[Access the Grafana user interface].
 
-. Click *Dashboards*, then *Import* to open the _Import Dashboard_ window and import the example Kafka Exporter dashboard (or paste the JSON).
+. Select the _Strimzi Kafka Exporter_ dashboard.
```
Comment (laidan6000) [inline documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc:29]: Use bold to refer to a UI element.

File: documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc [hunk 2]
```diff
@@ -23,20 +24,29 @@ If you are accessing the user interface for the first time, see xref:assembly-me
...
 +
 When metrics data has been collected for some time, the Kafka Exporter charts are populated.
 
-.Kafka Exporter Grafana charts
-
-From the metrics, you can create charts to display:
-
-* Message in per second (from topics)
-* Message in per minute (from topics)
+Strimzi Kafka Exporter:: Shows metrics for:
```
Comment (laidan6000) [inline documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc:33]: Is the repeated colon meant to be here?

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc [hunk 3]
```diff
@@ -6,53 +6,159 @@
...
-NOTE: No alert notification rules are defined.
+* `strimzi-kafka.json`
+* `strimzi-zookeeper.json`
+* `strimzi-kafka-connect.json`
+* `strimzi-kafka-mirror-maker-2.json`
+* `strimzi-operators.json`
 
-When accessing a dashboard, you can use the `port-forward` command to forward traffic from the Grafana pod to the host.
+The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
+You may need to modify the example dashboards or add other metrics, depending on your infrastructure.
```
Comment (laidan6000) [inline documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:19]: may > might

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc [hunk 4]
```diff
@@ -6,53 +6,159 @@
...
-
-. Click *Add* to test the connection to the data source.
+* Specify a Prometheus server URL (http://prometheus-operated:9090)
++
+Save and test the connection when you have added the details.
 +
 image::grafana_prometheus_data_source.png[Add Prometheus data source]
 
-. Click *Dashboards*, then *Import* to open the _Import Dashboard_ window and import the example dashboards (or paste the JSON).
+. From menu:Dashboards[Import], upload the example dashboards or paste the JSON directly.
```
Comment (laidan6000) [inline documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:77]: Are the `menu` and `btn` tags definitely rendered in the Strimzi docs?

File: documentation/modules/metrics/proc_metrics-deploying-grafana.adoc [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
 = Deploying Grafana
 
-To deploy Grafana to provide visualizations of Prometheus metrics, apply the xref:ref-metrics-config-files-{context}[example configuration file].
+To deploy Grafana to provide visualizations of Prometheus metrics, apply the `grafana.yaml` file provided in the `examples/metrics` directory.
```
Comment (scholzj) [inline documentation/modules/metrics/proc_metrics-deploying-grafana.adoc:9]: Maybe we should make it more clear that this is just one of the options and users should feel free to use their own Grafana installations?

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc [hunk 6]
```diff
@@ -6,53 +6,159 @@
...
+
+. Use `port-forward` to redirect the Grafana user interface to `localhost:3000`:
++
+[source,shell]
+----
+kubectl port-forward svc/grafana 3000:80
+----
 +
-On the initial view choose to reset the password.
+NOTE: If using the OpenShift web console, you can switch to menu:Networking[Routes], and click the URL for the `grafana` Route.
```
Comment (scholzj) [inline documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:57]: I would probably remove this. There are several other options how to do this on different platforms. Not sure it is worth to pick only one of them. Exposing it using routes is also questionable from security perspective.

File: documentation/modules/metrics/proc_metrics-deploying-grafana.adoc (hunk 5 above)
Comment (ppatierno) [reply documentation/modules/metrics/proc_metrics-deploying-grafana.adoc:9]: Which should be the same for the Prometheus installation as well.

File: documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc (hunk 2 above)
Comment (PaulRMellor) [reply documentation/modules/metrics/kafka-exporter/proc_kafka-exporter-enabling-grafana.adoc:33]: Yes.

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc (hunk 4 above)
Comment (PaulRMellor) [reply documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:77]: Good point. I built with asciidoctor and ccutil and seems okay. If they don't come out okay, I'll adjust

File: documentation/modules/metrics/proc_metrics-deploying-grafana.adoc (hunk 5 above)
Comment (PaulRMellor) [reply documentation/modules/metrics/proc_metrics-deploying-grafana.adoc:9]: To provide visualizations of Prometheus metrics,
you can use your own Grafana installation or deploy Grafana by applying the `grafana.yaml` file provided in the `examples/metrics` directory.

To obtain monitoring data from your Kafka cluster, you can use your own Prometheus deployment or deploy Prometheus by applying the xref:ref-metrics-config-files-{context}[example resource file for the Prometheus docker image and the YAML files for Prometheus-related resources].

File: documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc (hunk 6 above)
Comment (PaulRMellor) [reply documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc:57]: Okay

**Pull request merged**